src/components/ReportActionItem/ReportPreview.js

```diff
@@ -260,7 +260,10 @@ function ReportPreview(props) {
     }, [isPaidGroupPolicy, isCurrentUserManager, isDraftExpenseReport, isApproved, iouSettled]);
     const shouldShowSettlementButton = shouldShowPayButton || shouldShowApproveButton;
     return (
-        <OfflineWithFeedback pendingAction={lodashGet(props, 'iouReport.pendingFields.preview')}>
+        <OfflineWithFeedback
+            pendingAction={lodashGet(props, 'iouReport.pendingFields.preview')}
+            shouldDisableOpacity={props.action.pendingAction || props.action.isOptimisticAction}
+        >
```

Comment:
Just a confirmation question. 

Would it be better if `props.action.pendingAction || props.action.isOptimisticAction` was a variable such as `shouldDisableOpacity`?

Just like the `shouldShowSettlementButton`.